skt/executable.py

```diff
@@ -688,6 +688,7 @@ def main():
     args = parser.parse_args()
 
     setup_logging(args.verbose)
+    global cfg
```

Comment:
Let's not do this. Moreover, let's pass only the state that the particular class or function needs. We need them more independent , not more entangled.